calculators/gas_king_rating

```diff
@@ -0,0 +1,101 @@
+<?php
+
+$GLOBALS['rating_calculators']['gas_king_rating'] = [
```

Comment:
custom rating names start with _ (same for function prefixes which mimic rating name)

then follows the name of the author, then name

it should be something like "_pewpewpew_gas_king", or "_mingoubou_gas_king", or "_ppp_gasking", or "_noadkoko_gasking"...

Reply:
Should I only change it on this line or should my prefix also not contain "rating" and contain "_" before it ?

Reply:
your prefix must be the same as your rating name. Refer to _memoria_matchups.